launch/point_cloud.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
+  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
+
+  <node pkg="nodelet" type="nodelet" name="nodelet1"
+        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
+    <remap from="camera_info" to="/vision/front_cam/camera_info"/>
```

Comment:
Not sure whats going on here, is there more to the point cloud or does this do it?

Reply:
This is the launch file for the [depth image proc package](http://wiki.ros.org/depth_image_proc).